spec/rubocop/cop/style/it_block_parameter_spec.rb

```diff
@@ -0,0 +1,218 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Style::ItBlockParameter, :config do
+  context '>= Ruby 3.4', :ruby34, unsupported_on: :parser do
+    context 'EnforcedStyle: allow_named_parameter' do
+      let(:cop_config) { { 'EnforcedStyle' => 'allow_named_parameter' } }
+
+      it 'registers an offense when using a single numbered parameters' do
+        expect_offense(<<~RUBY)
+          block { do_something(_1) }
```

Comment:
Can you also add a test for `block { do_something(_2) }` (only the second argument is used)? Looking at the code, this should already be handled correctly.